### ISSUE TITLE
On breakpoint widget open the placeholder is not read

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141721]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev1; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Narrator;#NVDA; #ScreenReader; #WCAG4.2.1; #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1)Turn on Narrtor/NVDA
2)Open Visual studio code editor.
3)Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
4)Open the launch.json file under the "Configure or Fix: launch.json"button and select "Node.js" from the drop down list
5)Add configurations to "launch.json" utilizing IntelliSense with CTRL+SPACE
6)Parse launch.json with keyboard and observe if ATs will provide stock debugging information under "configurations" properties.
7)Start debugging by pressing F5.
8)Set a breakpoint and pause execution at the breakpoint by issuing an request/command to your Node process that triggers the 
   line you got the breakpoint
9)Toggle/add/remove breakpoints.
10)Verify whether Screen Reader is announcing the placeholder information while editing the condition or not.

### Actual Result:
Screen Reader is not announcing the placeholder information while editing the condition.

### Expected Result:
Screen Reader should announce the placeholder information while editing the condition.

### User Impact:
Screen reader user will not understand the content completely while editing the condition.
![On breakpoint widget open the placeholder is not read](https://user-images.githubusercontent.com/98530069/151558631-22e6e97c-bad7-4835-baee-c7f97dfbde8a.gif)

**Comment by isidorn** [comment]
@roblourens do not read above. Gist is: "on breakpoint widget open the placeholder is not read".
So try setting the aria-label on the editor title to be the placeholder. Hopefully that will get read. If that does not work feel free to use aria alert as plan B.

**Comment by roblourens** [comment]
Sorry what's the "placeholder" here?

**Comment by isidorn** [comment]
The dimmed text in the background of the widget: "Break when expression evaluates to true..."

![Screenshot 2022-01-31 at 10 37 53](https://user-images.githubusercontent.com/1926584/151770418-4e6a056e-982b-4527-b617-862173cf3622.png)
